primitives/domain/src/lib.rs

```diff
@@ -229,7 +229,7 @@ impl TryFrom<Vec<u8>> for MainchainPublicKey {
 pub const MAINCHAIN_ADDRESS_HASH_LEN: usize = 28;
 
 /// Some hash of MainchainAddress, 28 bytes. Presumably blake2b_224.
-/// Way to get it: cardano-cli address key-hash --payment-verification-key-file <path to vkey>
+/// Way to get it: cardano-cli address key-hash --payment-verification-key-file FILE
```

Comment:
`cargo doc` was thinking it is HTML marker. I've `FILE` instead. `cardano-cli` help message is using `FILE` as well.